projects/subject-access-requests-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/controller/SubjectAccessRequestsController.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.api.controller
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.service.SubjectAccessRequestsService
+
+@RestController
+@RequestMapping("probation-case/{crn}")
+@PreAuthorize("hasRole('PROBATION_API__SUBJECT_ACCESS_REQUEST___DETAIL')")
```

Comment:
Minor thing - think you have an extra `_` here:
```suggestion
@PreAuthorize("hasRole('PROBATION_API__SUBJECT_ACCESS_REQUEST__DETAIL')")
```

Reply:
done